packages/@lightningjs/ui-components/src/components/MetadataCardContent/MetadataCardContent.styles.js

```diff
@@ -19,6 +19,7 @@
 export const base = theme => ({
   detailsTextStyle: theme.typography.body3,
   descriptionTextStyle: { maxLines: 3 },
+  descriptionDetailsStyle: { paddingY: 5 },
```

Comment:
Perfect! Would you mind just also adding this to `MetadataCardContent.mdx` since this is a new style property?